frontend/controller/actions/group.js

```diff
@@ -13,23 +15,24 @@ import {
   PROPOSAL_INVITE_MEMBER,
   PROPOSAL_PROPOSAL_SETTING_CHANGE,
   PROPOSAL_REMOVE_MEMBER,
-  STATUS_OPEN,
-  STATUS_PASSED,
-  STATUS_FAILED,
-  STATUS_EXPIRING,
+  STATUS_CANCELLED,
   STATUS_EXPIRED,
-  STATUS_CANCELLED
+  STATUS_EXPIRING,
+  STATUS_FAILED,
+  STATUS_OPEN,
+  STATUS_PASSED
```

Comment:
Whatever setting you have enabled in your editor that changes the order of things for no reason, could you please disable it? 🙏